source/aggregation.txt

```diff
@@ -0,0 +1,190 @@
+.. _mongoid-aggregation:
+
+====================================
+Transform Your Data with Aggregation
+====================================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, transform, pipeline
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use {+odm+} to perform **aggregation
+operations**.
+
+Aggregation operations process data in your MongoDB collections and return
+computed results. The MongoDB Aggregation framework, which is part of the Query
+API, is modeled on the concept of data processing pipelines. Documents enter a
+pipeline that contains one or more stages, and this pipeline transforms the
+documents into an aggregated result. 
+
+An aggregation operation is similar to a car factory. A car factory has an assembly line, which contains assembly stations with specialized tools to do specific jobs, like drills and welders. Raw parts enter the factory, and then the assembly line transforms and assembles them into a finished product.
```

Comment:
S: I think the Rust docs present this info and the next section more cleanly https://github.com/mongodb/docs-rust/blob/master/source/fundamentals/aggregation.txt